drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -880,7 +908,17 @@ function DragAndDropBlock(runtime, element, configuration) {
                 }
             }
         }
-    }
+    };
+
+    var initializeSlider = function() {
+        if (pageLoaded) {
+            itemSlider = $root.find('.item-bank').bxSlider({
+                pager: false,
+                touchEnabled: false,
```

Comment:
Why disable touch?

Reply:
It was disabled just for testing purpose. I have enabled it.

Reply:
👍